tests/unittests/test_datasource/test_ovf.py

```diff
@@ -138,16 +139,29 @@ def test_get_data_false_on_none_dmi_data(self):
             'DEBUG: No system-product-name found', self.logs.getvalue())
 
     def test_get_data_no_vmware_customization_disabled(self):
-        """When vmware customization is disabled via sys_cfg log a message."""
+        """When cloud-init workflow for vmware is disabled via sys_cfgi and
```

Comment:
typo (sys_cfgi)

Reply:
Thanks, this is fixed and please take another look.